packages/app/src/TeamPage.tsx

```diff
@@ -38,6 +37,29 @@ export default function TeamPage() {
   const hasAllowedAuthMethods =
     team?.allowedAuthMethods != null && team?.allowedAuthMethods.length > 0;
 
+  const sentryDsn = useMemo(() => {
+    if (!team) {
+      return '';
+    }
+
+    try {
+      const ingestorUrl = new URL(process.env.NEXT_PUBLIC_INGESTOR_API_URL);
+
+      const sentryDsnParts = [
+        ingestorUrl.protocol,
+        '//',
+        team.apiKey.replace('-', ''),
+        '@',
+        ingestorUrl.host,
+        ingestorUrl.port ? `:${ingestorUrl.port}` : '',
+      ];
```

Comment:
It looks like the [Sentry DSN format](https://docs.sentry.io/product/sentry-basics/concepts/dsn-explainer/#the-parts-of-the-dsn) expects a project ID or else the package throws, we can just do `/1` at the end